superset/views/base_api.py

```diff
@@ -84,6 +91,31 @@ def __init__(self, field_name: str, filter_class: Type[BaseFilter]):
         self.filter_class = filter_class
 
 
+class BaseFavoriteFilter(BaseFilter):  # pylint: disable=too-few-public-methods
+    """
+    Base Custom filter for the GET list that filters all dashboards, slices
+    that a user has favored or not
+    """
+
+    name = _("Is favorite")
+    arg_name = ""
```

Comment:
We could even leverage this default and leave it blank in the subclasses :) (re: `favorited`)

Reply:
True, nice catch!

Reply:
I would create a BaseOuterJoinFilter with "where" and "on" arguments and then extend it

Reply:
That's interesting can you add more detail or an example?

Reply:
That's interesting can you add more detail or an example?